proposals/mail-helper-refactor.md

```diff
@@ -175,7 +175,7 @@ msg.bcc = [
 
 msg.header = Header('X-Test5', 'Test5', p=1)
 msg.header = Header('X-Test6', 'Test6', p=1)
-msg.headers = [
+msg.header = [
```

Comment:
This can be multiple headers, should it stay plural?

Reply:
I think it should stay plural in that case.